src/user/user.spec.js

```diff
@@ -80,6 +80,51 @@ describe("user", () => {
     });
   });
 
+  describe(".set() :", () => {
+    withData(
+      {
+        "sets an email": { email: "wayne_modified@vates.fr" },
+        "sets a password": { password: "newPassword" },
+        "sets a permission": { permission: "user" },
+      },
+      async data => {
+        data.id = await xo.createUser(simpleUser);
+        expect(await xo.call("user.set", data)).toBe(true);
+        expect(omit(await xo.getUser(data.id), "id")).toMatchSnapshot();
+
+        testConnection({
+          email: data.email === undefined ? simpleUser.email : data.email,
+          password:
+            data.password === undefined ? simpleUser.password : data.password,
+        });
+      }
+    );
+
+    it("fails trying to set the password of a nonexistant user", async () => {
+      await expect(
+        xo.call("user.set", {
+          id: "non-existent-id",
+          password: simpleUser.password,
+        })
+      ).rejects.toMatchSnapshot();
+    });
+
+    it.skip("fails trying to set an email already used", async () => {
+      await xo.createUser(simpleUser);
+      const userId2 = await xo.createUser({
+        email: "wayne6@vates.fr",
+        password: "batman",
+      });
+
+      await expect(
+        xo.call("user.set", {
+          id: userId2,
+          email: simpleUser.email,
+        })
+      ).rejects.toMatchSnapshot();
+    });
+  });
+
```

Comment:
Please add this tests:
- `A user cannot change its own permission`
- `A non admin user cannot set email, password, permission`

Reply:
I did these two, but If you have better names for tests, tell me.